sale_commission_type_lines/readme/USAGE.rst

```diff
@@ -0,0 +1,6 @@
+To use features of this module, you need to:
+
+#. Go to Sales > Commission Management > Commission Types.
+#. Create or update existing record.
+#. Create multiple rules. Select *Apply On* variant. Fill up other fields.
+#. Rest flow is according to OCA sale_commission module.
```

Comment:
To use features of this module, you need to:

#. Go to Sales > Commission Management > Commission Types.
#. Create a Commission Type with type = "Rules".
#. Create multiple rules based on variant/product/category or global
#. These rules will be sorted according to the same logic of sale pricelist.
#. Rest flow is according to OCA sale_commission module.